docs/WritingTests.asciidoc

```diff
@@ -284,8 +284,59 @@ Job dependencies are only resolved when using the iso controller to
 create new jobs from job templates. Posting individual jobs manually
 won't work.
 
-Job dependencies are currently only possible between tests that are
-scheduled for the same machine.
+Job dependencies now allow inter-machine dependency that make possible to create job dependencies between tests that are scheduled for the different machine.
+To use it, simply append machine for each dependent test suite with colon separated.
```

Comment:
Maybe add: If machine isn't explicitly defined, the var `%MACHINE%` of the current job is used.